src/main/java/com/google/sps/servlets/SearchServlet.java

```diff
@@ -86,8 +86,6 @@ public JSONObject getMatchingCourses(HttpServletRequest request) {
   private FilterPair getFilters(HttpServletRequest request, boolean fuzzy) {
     List<Filter> filters = new ArrayList<>();
     List<Filter> fuzzyFilters = new ArrayList<>();
-    // ArrayList<ArrayList<Filter>> filterLists = new ArrayList<ArrayList<Filter>>();
```

Comment:
remove comment

Reply:
Doesn't the red highlight mean I removed this line?